easybuild/easyconfigs/g/GtkSourceView/GtkSourceView-3.24.11-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'GtkSourceView'
+version = '3.24.11'
+
+homepage = 'https://wiki.gnome.org/Projects/GtkSourceView'
+description = """
+ GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK+
+ widget for multiline text editing. GtkSourceView adds support for syntax
+ highlighting, undo/redo, file loading and saving, search and replace, a
+ completion system, printing, displaying line numbers, and other features
+ typical of a source code editor.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('pkg-config', '0.29.2'),
+    ('GObject-Introspection', '1.60.1', '-Python-3.7.2'),
+]
+
+dependencies = [
+    ('GLib', '2.60.1'),
+    ('GTK+', '3.24.8'),
+    ('libxml2', '2.9.8'),
+]
+
+configopts = "--disable-silent-rules --enable-introspection=yes "
```

Comment:
@Flamefire just a question: why don't you include these options in version 4.4.0 as well?

Reply:
Because this uses ConfigureMake, the other Meson